app/controllers/applications_controller.rb

```diff
@@ -29,8 +30,18 @@ def create
       redirect_to "/applications/new"
       flash[:alert] = "Error: #{error_message(applicant.errors)}"
     end
+  end
 
-
+  def update
+    application = PetsApplication.find(params[:id])
+    if application.pet_id == nil
+      application.update ({
+        pet_id: params[:pet]
+      })
+    else
+      PetsApplication.create!(applicant_id: application.applicant_id, pet_id: params[:pet])
+      redirect_to "/applications/#{application.applicant_id}"
+    end
```

Comment:
update method looks good. should allow us to add a pet to an application, as well as create a new PetsApplication row in the case an application already has a pet.